.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    pull_request:
+      branches: 
+        - '**'
+
+jobs:
+  build:
+
+    env:
+      APP_ENV: test
+    
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        php: [ '7.3', '7.4' ]
+
+    runs-on: ${{matrix.os}}
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    
+    - name: Set up PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{matrix.php}}
+        coverage: none
+
+    - name: Install wikimedia-ocr
+      run: |
+        composer install
+        npm run build
+
+    - name: Test
+      run: |
+        composer test
+        ./bin/phpunit --exclude-group=exclude-from-ci
```

Comment:
We don't yet have phpunit in use, so this will error. Also, `exclude-from-ci` is only something we do in WS Export because we haven't yet got all the tests passing; it's probably not something we'll aim to do here.

Reply:
Done.